docs/source/release/v3.12.0/diffraction.rst

```diff
@@ -41,6 +41,9 @@ Engineering Diffraction
 Single Crystal Diffraction
 --------------------------
 - :ref:`FilterPeaks <algm-FilterPeaks>` now supports filtering peaks by TOF, d-spacing, and wavelength.
+- HB3A reduction interface has been enhanced.  A child window is added to it for users to pre-process scans and save the processed and merged data to NeXus files in order to save time when they start to reduce and visualize the data.
```

Comment:
Is this saying the same thing as line 48?

Reply:
I think you are correct. Probably came in when merging. I will remove the duplicate.